### PR TITLE
fixed the initialization of javascript classes (fixes #unic/estatico/…

### PR DESCRIPTION
**Pull request by swey**
…issues/35) + fixed the duplicated values in initEvents, which resulted in multiple inits of the same module

### CODE REVIEW

File: source/modules/.scaffold/scaffold.hbs [hunk 1]
```diff
@@ -1,3 +1,3 @@
-<div class="mod_{{name}}" data-init="{{className}}">
+<div class="mod_{{name}}" data-init="{{keyName}}">
```
Comment (swey) [inline source/modules/.scaffold/scaffold.hbs:1]: See #35

File: source/assets/js/helpers/estaticoapp.js [hunk 2]
```diff
@@ -65,6 +65,9 @@ class EstaticoApp {
 			};
 
 			this.initEvents = this.initEvents.concat(Module.initEvents);
+
+			// Remove duplicates from initEvents
+			this.initEvents = [...new Set(this.initEvents)];
```
Comment (swey) [inline source/assets/js/helpers/estaticoapp.js:70]: Without this addition the `initEvents` array is getting longer and longer and has several times the same values, which result in multiple inits of the modules.

(An alternative is a merge of `this.initEvents` and `Module.initEvents`.)

Comment (backflip) [reply source/assets/js/helpers/estaticoapp.js:70]: ES6 is awesome! Didn't know about `new Set` functionality until today

**Comment by backflip** [comment]
LGTM

File: source/assets/js/helpers/estaticoapp.js (hunk 2 above)
Comment (swey) [reply source/assets/js/helpers/estaticoapp.js:70]: Yeah, tried to avoid using any custom merge functionality ;)

Comment (backflip) [reply source/assets/js/helpers/estaticoapp.js:70]: Do you know by any chance what this transpiles to (with our babel settings)?

Comment (backflip) [reply source/assets/js/helpers/estaticoapp.js:70]: Never mind: https://babeljs.io/repl/#?babili=false&evaluate=true&lineWrap=false&presets=es2015&targets=&browsers=&builtIns=false&code=let%20bla%20%3D%20%5B...new%20Set(bla)%5D%3B

**Review by orioltf — APPROVED** [review]

**Comment by orioltf** [comment]
Thanks so much for so many PRs! :)

**Comment by swey** [comment]
Welcome! More to come :)

**Pull request merged**